index.html

```diff
@@ -2389,6 +2389,7 @@ <h5 class="contact-head">Contact Us</h5>
     <script src="https://cdn.jsdelivr.net/npm/axios/dist/axios.min.js"></script>
     <script type="text/javascript" src="js/lity.js"></script>
     <script type="text/javascript" src="//buttons.github.io/buttons.js" async defer></script>
+    <script type="text/javascript" src="js/diaries-data.js"></script>
```

Comment:
We are displaying images on almost all of the website pages. Why did you include it on the homepage only?